packages/eslint-config-airbnb-base/rules/style.js

```diff
@@ -3,531 +3,823 @@ module.exports = {
+    //
+    // const bleh = {
+    //   someProp:      1
+    //   someOtherProp: 2
+    // }
+    //
+    // or even
+    //
+    // const bleh = {
+    //   someProp      : 1
+    //   someOtherProp : 2
+    // }
+    //
+    // I'm personally a fan of the second option, but this should probably be discussed by the group to see what everyone's preferences are
     'key-spacing': ['error', { beforeColon: false, afterColon: true }],
 
     // require a space before & after certain keywords
-    'keyword-spacing': ['error', {
-      before: true,
-      after: true,
-      overrides: {
-        return: { after: true },
-        throw: { after: true },
-        case: { after: true }
-      }
-    }],
+    // KEEP
+    'keyword-spacing': [
+      'error',
+      {
+        before: true,
+        after: true,
+        overrides: {
+          return: { after: true },
+          throw: { after: true },
+          case: { after: true },
+        },
+      },
+    ],
 
     // enforce position of line comments
     // https://eslint.org/docs/rules/line-comment-position
     // TODO: enable?
-    'line-comment-position': ['off', {
-      position: 'above',
-      ignorePattern: '',
-      applyDefaultPatterns: true,
-    }],
+    // KEEP, leave it off
+    'line-comment-position': [
+      'off',
+      {
+        position: 'above',
+        ignorePattern: '',
+        applyDefaultPatterns: true,
+      },
+    ],
 
     // disallow mixed 'LF' and 'CRLF' as linebreaks
     // https://eslint.org/docs/rules/linebreak-style
+    // KEEP, although we may want to discuss whether we want to use CR or CRLF (at least until we all get MacBooks and then we can go unix style :D )
     'linebreak-style': ['error', 'unix'],
 
     // require or disallow an empty line between class members
     // https://eslint.org/docs/rules/lines-between-class-members
-    'lines-between-class-members': ['error', 'always', { exceptAfterSingleLine: false }],
+    // KEEP
+    'lines-between-class-members': [
+      'error',
+      'always',
+      { exceptAfterSingleLine: false },
+    ],
 
     // enforces empty lines around comments
+    // KEEP, leave it off
     'lines-around-comment': 'off',
 
     // require or disallow newlines around directives
     // https://eslint.org/docs/rules/lines-around-directive
-    'lines-around-directive': ['error', {
-      before: 'always',
-      after: 'always',
-    }],
+    // DISABLE: we don't use any directives (ie. 'use strict' or 'use asm'). We can enable strict mode at the TS compiler level via the tsconfig
+    'lines-around-directive': [
+      'error',
+      {
+        before: 'always',
+        after: 'always',
+      },
+    ],
 
     // specify the maximum depth that blocks can be nested
+    // SITUATIONAL ANALYSIS NEEDED: Deep levels of nesting to me is a code smell that suggests the logic should be broken up into smaller functions, but
+    // this should warrant a discussion with the team
     'max-depth': ['off', 4],
```

Comment:
SITUATIONAL ANALYSIS NEEDED: Deep levels of nesting to me is a code smell that suggests the logic should be broken up into smaller functions, but this should warrant a discussion with the team.